### DISCUSSION TITLE
Access eclipse jars mid build

### DISCUSSION THREAD

**Discussion by rcspirent** [eclipse-tycho/tycho#259] [Q&A]
We have a multistage eclipse product build. The "target platform" was previously a pre-built eclipse-derivative.
Via eclipse-antrunner and a bunch of eclipse jars (from the plugins directory) source code was generated.

Now we're trying to replicate this using eclipse-run to start antrunner including a switch to target platform proper.
We want to synchronize the used jars to the target platform.
The issue here is that the eclipse-run eclipse only references them from the .m2 repository and does not put them in the plugins  directory.

Now we're wondering if there's a good way to get the paths to the used plugins/jars per target platform.
One way we came up with would be to parse the eclipse-run config.ini.

**Comment by akurtakov** [comment]
You should try https://github.com/eclipse/tycho/blob/master/RELEASE_NOTES.md#mixed-reactor-build-support in the upcoming 2.7.0 release.
